### ISSUE TITLE
[Application Request] TDARR node

### ISSUE THREAD

**Issue by RinnosukeETQW** [GhostWriters/DockSTARTer#1256]
# Name of the application
TDARR_Node
## GitHub
https://github.com/HaveAGitGat/Tdarr
## Docker Hub/GitHub Registry
haveagitgat/tdarr_node (only populated on preview right now)
### Additional context
Tdarr just released a complete rewrite for their v2 today, needs both a node as well as a server now. Existing docker repo will change to this version on 2021/01/31 and dockSTARTer's existing setup should only do the server version. In the meantime I'm hoping to get the basics in place so I can get the preview version of the node in place

**Comment by angrycuban13** [comment]
What is the difference between the server and the node? I don't use Tdarr so not super familiar with the inner workings of it.

**Comment by RinnosukeETQW** [comment]
The Server contains the DB for everything while the node actually does the
transcodes. It works where you can actually have nodes on
separate computers from the server, I have it running on my gaming machine
now to take advantage of the video card

**Comment by angrycuban13** [comment]
Someone ELI5'd me. Makes sense, I just don't use it that often so wanted to know. 

I'll look into it. Feel free to ping me if you don't see a PR in 3-5 days :)

**Comment by angrycuban13** [comment]
Also if you have a compose override for it to go with the already included Tdarr it won't hurt to show me!

**Comment by angrycuban13** [comment]
@RinnosukeETQW no need to close it. I automatically closed them as PRs get merged

**Comment by RinnosukeETQW** [comment]
Must have accidently done that when trying to reply, sorry.

**Comment by angrycuban13** [comment]
Yes if you have an override that'd be great :)

**Comment by RinnosukeETQW** [comment]
I was going to get you the actual setup but my zfs pool just died on me, I only overided the container with haveagitgat/tdarr:v2-preview though. Everything else worked.

**Comment by angrycuban13** [comment]
Sounds good!

**Comment by RinnosukeETQW** [comment]
Ok, got my server up and running again (turns out it was the drive sled for one of my drives not wanting to work) the override I have in place is as follows, but I'm sure you could tell from the container I listed

  tdarr:
     image:  haveagitgat/tdarr:v2-preview
     
   As I said before though this SHOULD change today when v2 goes into the main repo that should already be setup. Also worth noting that I'm a discord subscriber of his and this gives me access to better repos.

**Comment by RinnosukeETQW** [comment]
Just dug a little more, my entire library was wiped when I ran a sudo ds -c to update things, they shifted to their own db instead of MongoDB, this change is reflected for docker in "-v /home/h/Desktop/server:/app/server \" on their example, not sure the way to change this for dockstarter, but worth looking at.

**Comment by angrycuban13** [comment]
Yeah if you look at my PR I updated a the actual Tdarr app to point to the new container paths.
